### PR TITLE
🌱 Fix doc references to NodeCIDR in v1alpha8

### PR DESCRIPTION
**Pull request by mdbooth**
Fixes: #1887

/hold

### CODE REVIEW

**Comment by mdbooth** [comment]
@MaysaMacedo Please can you take a look at this, particularly the new doctext for Subnets?

File: config/crd/bases/infrastructure.cluster.x-k8s.io_openstackclusters.yaml [hunk 1]
```diff
@@ -5534,8 +5535,12 @@ spec:
                     type: string
                 type: object
               subnets:
-                description: If NodeCIDR cannot be set this can be used to detect
-                  existing IPv4 and/or IPv6 subnets.
+                description: |-
+                  Subnets specifies existing subnets to use if not ManagedSubnets are
+                  specified. All subnets must be in the network specifed by Network.
```
Comment (huxcrux) [inline config/crd/bases/infrastructure.cluster.x-k8s.io_openstackclusters.yaml:5540]: Just to make the lint happy :)
```suggestion
                  specified. All subnets must be in the network specified by Network.
```

Comment (mdbooth) [reply config/crd/bases/infrastructure.cluster.x-k8s.io_openstackclusters.yaml:5540]: Thank you! Committed and squashed.

**Comment by pierreprinetti** [comment]
Thank you @mdbooth !

**Pull request merged**